streaming/consumer.cc

```diff
@@ -11,13 +11,42 @@
 #include "consumer.hh"
 #include "replica/database.hh"
 #include "mutation/mutation_source_metadata.hh"
+#include "db/config.hh"
 #include "db/view/view_update_generator.hh"
 #include "db/view/view_update_checks.hh"
 #include "sstables/sstables.hh"
 #include "sstables/sstables_manager.hh"
 
 namespace streaming {
 
+sstables::offstrategy maybe_disable_offstrategy(const replica::database& db, const sstables::compaction_strategy& cs, const schema_ptr& s, sstables::offstrategy offstrategy) {
```

Comment:
Why do you need `offstrategy` parameter which is only used as a returned value AFAICT.
What am I missing here?

Reply:
NM, I got it.